delivering/release-process/webapp-release-process.md

```diff
@@ -9,28 +9,28 @@ title: Web Applications
 
 ### Individual Engineer
 
-As an engineer you have responsibilities to ensure the work you've done is reliably delivered to production:
+As an engineer you have responsibilities to ensure the work you've done is reliably released into production:
 
 
 - Before your code is merged:
   - Coordinate with DevOps on having configuration in place both on RC and production for your changes _before_ it gets merged so that releases are not held up.
 - After your code is merged:
-  - When the changes have been deployed on RC, verify your changes with a round of functional testing.
+  - When the changes have been deployed on RC, verify your changes via functional testing and obtain acceptance from any necessary stakeholders.
 - After your code is released:
-  - Perform smoke testing of the functionality if it is prudent and won't disrupt other users.
+  - Perform smoke testing of the functionality if it won't interfere other application users.
 
 ### Engineering Team
 
-Collectively the team members who have made changes are responsible for coordinating the release to production:
+Collectively, the team members who have made changes are responsible for coordinating the release to production:
 
-- The team members who have changes in the release candidate are responsible for verifying their changes:
+- The team members who have changes to the main branch are responsible for verifying their changes:
```

Comment:
Why did you make this change? It seems to me only those with commits in the RC can verify their changes.

Reply:
the documentation does not provided a clear explanation that "release candidate" is referring to a branch within Github and not the release itself.